straxen/analyses/posrec_comparson.py

```diff
@@ -0,0 +1,89 @@
+import numpy as np
+import pandas as pd
+
+import strax
+import straxen
+
+export, __all__ = strax.exporter()
+
+@straxen.mini_analysis(requires=('event_basics', 'event_posrec_many',))
+def load_corrected_positions(context, run_id, events,
+                             cmt_version='ONLINE',
+                             posrec_algos=['mlp', 'gcn', 'cnn']):
```

Comment:
Can you make this a tuple? List as arguments are:
[the source of all evil](https://florimond.dev/blog/articles/2018/08/python-mutable-defaults-are-the-source-of-all-evil/)

Reply:
Done 👍